### ISSUE TITLE
will there be a new version?

### ISSUE THREAD

**Issue by graph** [albertodemichelis/squirrel#214]
3.1 is 4 years ago. I see there are alot of good new fixes. I like having version numbers as I can have all my projects synchronized and it feels better for me to do that. A simple scheme of new version after maybe a few months or as it makes sense. If the intention is to use master branch it will make me feel better if the project says so somewhere. What is the plan & recommendations? Can we have newer official versions released?

Thank you.

**Comment by albertodemichelis** [comment]
Yes, I guess I could bless the current master as 3.2. It as been in production for a while now.

**Comment by rversteegen** [comment]
If you want to declare version 3.2, could you update the website? It still says that the current version is 3.0.7!

**Comment by D22699** [comment]
Squirrel is very easy to use~~~

Thanks for the development of @ Alberto Demichelis! Only after version 3.1 was updated in 2016, there was no news again?



Recently, the new features of C + + 23 standard have been added a lot? Ranges has the most features, and ranges has become the biggest winner. Our question is, can we integrate and bind the ranges syntax above C + + 20 in squirrel? Because it's so convenient to write business code with ranges!



Ranges realizes a variety of parallel combined filtering operations and other operations through pipeline symbols, which should also be the core functional goal of the next generation of squirrel. It not only saves a lot of code, but also greatly increases the flexibility of the algorithm!



In addition, can squirrel support the isolation of multi sub table data and operations? We hope that on the basis of the root table, there can be completely independent sub tables, which can support the same name of objects, and the algorithm can be limited to a certain sub table. C + + 20 provides modules, realizes a high degree of isolation, and even isolates Hongdu? Is there a similar feature in JS? For example, if we declare in the root table and instantiate in different sub tables, the algorithm and object will not be afraid of the problem of duplicate names, and the data will be effectively isolated? In this way, it is convenient to write a large amount of business code.




We are highly optimistic about squirrel and hope @ Alberto Demichelis can give an answer!



thank! thank! thank!
